ChangeLog

```diff
@@ -1,3 +0,0 @@
-
-	ChangeLog
-	
```

Comment:
Don't delete ChangeLog.

Reply:
ChangeLog was not deleted, It became a MarkDown file

Reply:
Doesn't that mean you have to update the Makefile too?
When I'm running `sudo make install` I'm getting:

`cd . && /bin/bash /home/rossi-ubuntu/Files/snaskii/snaskii22/build-aux/missing automake-1.16 --gnu
Makefile.am: error: required file './ChangeLog' not found
make: *** [Makefile:347: Makefile.in] Error 1`